src/metabase/models/permissions.clj

```diff
@@ -1079,6 +1079,33 @@
     (when (and has-native-perms? (not (group-has-native-perms? group-or-id database-or-id)))
       (grant-native-readwrite-permissions! group-or-id database-or-id))))
 
+; Audit Permissions helper fns
+
+(defenterprise default-audit-db-id
+  "OSS implementation of `audit-db/default-audit-db-id`, which is an enterprise feature, so does nothing in the OSS
+  version."
+  metabase-enterprise.audit-db [] ::noop)
+
+(defenterprise default-audit-collection-entity-id
+  "OSS implementation of `audit-db/default-audit-collection-entity-id`, which is an enterprise feature, so does nothing in the OSS
+  version."
+  metabase-enterprise.audit-db [] ::noop)
+
+(defn check-audit-db-permissions
+  "Check that the changes coming in does not attempt to change audit database permission. Admins should
+  change these permissions in application monitoring permissions."
+  [changes]
+  (let [changes-ids (->> changes
+                         vals
+                         (map keys)
+                         (apply concat))]
+    (when (some #{(default-audit-db-id)} changes-ids)
+      (throw (ex-info (tru
+                       (str "Unable to update audit database, that requires updating through monitoring permissions."))
```

Comment:
nit — accuracy (no longer using monitoring perms) + rewording suggestion but feel free to change it further
```suggestion
                       (str "Audit database permissions can only be changed by updating audit collection permissions."))
```